easybuild/easyconfigs/b/BDBag/BDBag-1.8.0-foss-2025a.eb

```
@@ -0,0 +1,34 @@
+# Author: Franky Backeljauw (Universiteit Antwerpen)
+
+easyblock = 'PythonBundle'
+
+name = 'BDBag'
+version = '1.8.0'
+
+homepage = 'https://github.com/fair-research/bdbag'
+description = """The bdbag utilities are a collection of software programs for
+working with BagIt packages that conform to the Bagit and Bagit/RO profiles."""
+
+toolchain = {'name': 'foss', 'version': '2025a'}
+
+dependencies = [
+    ('Python', '3.13.1'),
+    ('Python-bundle-PyPI', '2025.04'),  # for requests, certifi and pytz
```

adding a stupid change to trigger a sync..
```suggestion
    ('Python-bundle-PyPI', '2025.04'),  # for requests, certifi and pytz.
```